File: 프로그래머스/1/42748. K번째수/K번째수.js
function solution(array, commands) {
    let answer = [];
    for(let i = 0; i < commands.length; i++){
        let sliceArray = array.slice(commands[i][0] - 1, commands[i][1]);
        sliceArray = sliceArray.sort((a, b) => a- b);
        answer[i] = sliceArray[commands[i][2] -1]
    }
    return answer;
}

File: 프로그래머스/1/42748. K번째수/K번째수.py
def solution(array, commands):
    answer = []
    for i in range(len(commands)) :
        tmp_arr = array[commands[i][0]-1 : commands[i][1]]
        sort_arr = sorted(tmp_arr)
        answer.append(sort_arr[commands[i][2] - 1])        
    return answer